templates/README.md.gotmpl

```diff
@@ -198,12 +198,36 @@ In general, PRs are welcome. We follow the typical "fork-and-pull" Git workflow.
 
  1. **Fork** the repo on GitHub
  2. **Clone** the project to your own machine
- 3. **Commit** changes to your own branch
- 4. **Push** your work back up to your fork
- 5. Submit a **Pull Request** so that we can review your changes
+ 3. [**Setup pre-commit**](####) to verify code style
+ 4. **Commit** changes to your own branch
+ 5. **Push** your work back up to your fork
+ 6. Submit a **Pull Request** so that we can review your changes
 
 **NOTE:** Be sure to merge the latest changes from "upstream" before making a pull request!
 
+#### Setup
```

Comment:
```suggestion
#### Developer Setup
```

Reply:
Consider only including this if there's a `.pre-commit-config.yaml` file.

https://github.com/cloudposse/terraform-example-module/blob/master/.pre-commit-config.yaml